hypothesis-python/scripts/basic-test.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e -o xtrace
 
-cd "$(dirname "$0")"/../hypothesis-python
+cd "$(dirname "$0")"/..
```

Comment:
Is this meant to be cd'ing to the directory this script lives in?

If so, would https://stackoverflow.com/a/246128/1558022 potentially be more robust? (e.g. if this script is called from something else, is `$0` set correctly?)

Reply:
@DRMacIver the 👍 makes me think you wanted to take this suggestion, but it looks like you didn't?

(I don't have any opinion and definitely haven't read that SO answer. Just checking in to see if there's something you wanted to do and didn't here.)

Reply:
Whoops! I did it in a bunch of places but not the one that @alexwlchan actually commented on. Thanks.